ci/misbehaviour/misbehaviour_test.sh

```diff
@@ -55,7 +55,7 @@ cat "$HERMES_LOG"
 info "--------------------------------------------------"
 echo ""
 
-if grep -q "Evidence succesfully submitted" "$HERMES_LOG"; then
+if grep -q "Evidence successfully submitted" "$HERMES_LOG"; then
```

Comment:
This fix seems to have surfaced an issue with this assertion.
Could you update it in the following way. Since it was matching the incorrect string this issue was hidden.
```
if grep -q "Evidence successfully submitted" "$HERMES_LOG"; then
    info "Misbehaviour detected and submitted successfully!"
else
    warn "Misbehaviour detection failed!"
    exit 1
fi
```